README.md

```diff
@@ -1,65 +1,93 @@
-# Welcome to Streamlit :wave:
+![Streamlit logo|200x200](https://i.ibb.co/5FdSr1V/streamlit.png)
 
-**The fastest way to build and share data apps.**
+# Streamlit
 
-Streamlit lets you turn data scripts into shareable web apps in minutes, not weeks. It’s all Python, open-source, and free! And once you’ve created an app you can use our [Community Cloud platform](https://streamlit.io/cloud) to deploy, manage, and share your app!
+**A faster way to build and share data apps.**
+
+Streamlit lets you turn data scripts into shareable web apps in minutes, not weeks. It’s all Python, open-source, and free! And once you’ve created an app you can use our [Community Cloud platform](https://streamlit.io/cloud) to deploy, manage, and share your app!
 
-![Example of live coding an app in Streamlit|635x380](https://raw.githubusercontent.com/streamlit/docs/main/public/images/Streamlit_overview.gif)
 
 ## Installation
 
+Open a terminal and run:
 ```bash
-pip install streamlit
-streamlit hello
+$ pip install streamlit 
+$ streamlit hello
 ```
 
-Streamlit can also be installed in a virtual environment on [Windows](https://github.com/streamlit/streamlit/wiki/Installing-in-a-virtual-environment#on-windows), [Mac](https://github.com/streamlit/streamlit/wiki/Installing-in-a-virtual-environment#on-mac--linux), and [Linux](https://github.com/streamlit/streamlit/wiki/Installing-in-a-virtual-environment#on-mac--linux).
+This should open our sweet _Streamlit Hello_ app right in your browser. It features a bunch of examples of what you can do with Streamlit!
 
-## A little example
+<img src="https://media.cleanshot.cloud/media/55064/9RAuEZKubvrlrlPVua50xuhyPIiD8MBdvzi7Pa3s.gif?Expires=1674790062&Signature=fqIRLW0tMXntC1vCrSq3yLPTtA58SRLiO1i0y~7YshYuOfQQokc0OI6uxWg6IBL8rw2gGnzA4ML-OdGWPgK6WSulyLR-wNbQAPiKX4LF0vXS9F6AmZ8V39Tze5Fo9TWfA3JZF3mnu7q6JrkssCk~6614xuugx9gojkOJMtsbFrOiM2kNLkje2OepBbxromYKVLLEQGwuPcFvUiA3Hb6ZuSHtNX~99wLu94PAKpXfPUf21sCt4vfmYItkl0z6BuHWYP5Zs5SlvPo8iXvdrb-dKL-~-kbTRjpcTFgWhFT1zg8iNZNCeR0Q~QvpS8rzfQPMMeIiBiRJFmTm4RC7~FRktA__&Key-Pair-Id=K269JMAT9ZF4GZ" alt="Streamlit Hello" width=500 href="none"></img>
```

Comment:
Instead of the normal Chrome window, can you put this into the simplified browser window we use on the blog? See e.g. the images in [here](https://blog.streamlit.io/create-a-color-palette-from-any-image/). Seems like we have one version with gray controls and one with colored controls. Not sure which one is better, probably worth just trying out both. But in principle, I think color doesn't hurt here.

Reply:
Also, super tiny nit but the anchor thing that shows up on hover bothers me :D

Reply:
- Changed that into a GIF!
- Can't use the image-generator.streamlit.app app to generate GIFs unfortunately. My GIFs skills stop here ;-) feel free to edit them if you have a solution
- I don't see what you mean with the anchor thing?